easybuild/easyconfigs/m/Mikado/Mikado-2.3.4-foss-2022b.eb

```diff
@@ -0,0 +1,85 @@
+easyblock = 'PythonBundle'
+
+name = 'Mikado'
+version = '2.3.4'
+
+homepage = 'https://github.com/EI-CoreBioinformatics/mikado'
+description = """Mikado is a lightweight Python3 pipeline to identify
+ the most useful or “best” set of transcripts from multiple transcript
+ assemblies. Our approach leverages transcript assemblies generated
+ by multiple methods to define expressed loci, assign a representative
+ transcript and return a set of gene models that selects against transcripts
+ that are chimeric, fragmented or with short or disrupted CDS."""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+    ('Ninja', '1.11.1'),
+    ('Meson', '0.64.0'),
+]
+
+dependencies = [
+    ('Python', '3.10.8'),
+    ('Biopython', '1.81'),
+    ('hypothesis', '6.68.2'),
+    ('networkx', '3.0'),
+    ('SciPy-bundle', '2023.02'),
+    ('Pysam', '0.21.0'),
+    ('PyYAML', '6.0'),
+    ('snakemake', '7.32.3'),
+    ('pyfaidx', '0.7.2.1'),
+    ('Greenlet', '2.0.2'),
```

Comment:
add `drmaa-python` as commented out dependency, along with a comment to explain why it can be useful, and why it's commented out